Makefile

```diff
@@ -36,6 +36,9 @@ requirements:
 bdist_wheel: requirements
 	$(PYTHON) setup.py bdist_wheel
 
+sdist: requirements
+	$(PYTHON) setup.py sdist
```

Comment:
We also deploy source on Azure. Do we want to drop one or keep both?

Reply:
I don't think we do:

```yaml
  - script: python setup.py bdist_wheel
    displayName: 'Building artifact'
```

We build the whl file and then deploy all contents of `dist/`, which will only contain the wheel.

Reply:
Oh you're right, I'm looking at `master`, which is behind... 🤦‍♂